src/app/TERMS.md

```diff
@@ -0,0 +1,252 @@
+- Remove any copyright, trademark or other proprietary rights notice from the Site or Services;
+- Frame or mirror any portion of the Site or Services, or otherwise incorporate any portion of the Site into any product or service, without DeXter's express prior written consent;
+- Restrict, discourage or inhibit any person from using the Site or Services;
+- Systematically download and store content from the Site or Services;
+- Use any robot, spider, site search/retrieval application or other manual or automatic device to retrieve, index, “scrape,” “data mine”, or otherwise gather content on the Site or Services (including Submissions), or reproduce or circumvent the navigational structure or presentation of the Site, without DeXter's express prior written consent. Notwithstanding the foregoing, and subject to compliance with applicable law and any instructions posted in the robots.txt file located in the Site’s root directory, DeXter grants to the operators of public search engines permission to use spiders to copy materials from the Site for the sole purpose of (and solely to the extent necessary for) creating publicly available, searchable indices of such materials, but not caches or archives of such materials. DeXter reserves the right to revoke such permission either generally or in specific cases, at any time and without notice.
+- Take any action that imposes an unreasonable or disproportionately large load on our infrastructure or that of our Nodes, or detrimentally interfere with, intercept, or expropriate any system, data or information;
+- Post, transmit or otherwise make available through or in connection with the Site or Services any virus, worm, Trojan horse, Easter egg, time bomb, spyware, ransomware, malware, or other computer code, file or program that is or is potentially harmful or invasive or intended to damage or hijack the operation of, or to monitor the use of, any hardware, software or equipment.
+- Otherwise attempt to gain unauthorized access to or use of the Site, the API, other Accounts, Nodes, or computer systems connected to the Site; or
+- Engage in any behavior which violates this Agreement or is otherwise deemed unacceptable by us in our sole discretion.
+
+## Reservation of Rights
+
+DeXter reserves the right to bar any transactions on the Site, for or with, any user with or without cause, at any time, subject to any limitations imposed by applicable law. In lieu of refusing access to the Site or Services, DeXter may, in its sole and absolute discretion, perform due diligence. You may be subject to due diligence procedures in your use of the Site or Services. If you decline to provide requested information or otherwise do not reply timely or substantively with the documentation or data requested, DeXter has the absolute discretion to immediately suspend or terminate your Account.
```

Comment:
Do we actually have a means of preventing access to a user?  I presume we could block an account address from connecting, but suspend and termination doesn't seem viable